aws_quickstart/datadog_api_call_v2.py

```diff
@@ -0,0 +1,127 @@
+import boto3
+
+import json
+import logging
+import signal
+from urllib.request import build_opener, HTTPHandler, Request
+import urllib.parse
+
+LOGGER = logging.getLogger()
+LOGGER.setLevel(logging.INFO)
+
+def call_datadog_api(event, method):
+    api_key = event['ResourceProperties']['APIKey']
+    app_key = event['ResourceProperties']['APPKey']
+    api_url = event['ResourceProperties']['ApiURL']
+    account_id = event['ResourceProperties']['AccountId']
+    role_name = event['ResourceProperties']['RoleName']
+    host_tags = event['ResourceProperties']['HostTags']
+    cspm = event['ResourceProperties']['CloudSecurityPostureManagement']
+    metrics_enabled = event['ResourceProperties']['DisableMetricCollection']
+
+    # Make the url Request
+    url = 'https://api.' + api_url + '/api/v1/integration/aws'
+    values = {
+        'account_id': account_id,
+        'role_name': role_name,
+    }
+    if method != "DELETE":
+        values["host_tags"] = host_tags
+        values["cspm_resource_collection_enabled"] = cspm == "true"
+        values["metrics_collection_enabled"] = metrics_enabled == "false"
```

Comment:
This looks wrong but I think it's actually right since the polarity of `metrics_enabled` is wrong

Reply:
Yeah, I misnamed that variable.